hibernate-core/src/main/java/org/hibernate/internal/util/xml/XMLHelper.java

```diff
@@ -59,6 +59,14 @@ public DocumentFactory getDocumentFactory() {
 
 	public SAXReader createSAXReader(ErrorLogger errorLogger, EntityResolver entityResolver) {
 		SAXReader saxReader = new SAXReader();
+		try {
+			saxReader.setFeature( "http://apache.org/xml/features/nonvalidating/load-external-dtd", false );
+			saxReader.setFeature( "http://xml.org/sax/features/external-general-entities", false );
+			saxReader.setFeature( "http://xml.org/sax/features/external-parameter-entities", false );
```

Comment:
please add line here also:
`saxReader.setFeature( "http://xml.org/sax/features/external-parameter-entities", false );  // Do not include external parameter entities or the external DTD subset.  `

Suggestion is because the comment gives me an idea of what the property settings accomplish.